README.md

```diff
@@ -2,23 +2,75 @@
+`cri-containerd` is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+The current release of `cri-containerd` is `v0.1.0-alpha.1`. This release is for
+use with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Getting started
+### Binary Dependencies and Specifications
+The alpha release of cri-containerd has been tested with:
+* [containerd](https://github.com/containerd/containerd) v0.2.3-1098-g8ed1e24
+* [runc](https://github.com/opencontainers/runc) v1.0.0-rc3
+* [CNI](https://github.com/containernetworking/cni) v0.4.0
+
+As containerd and runc move to their respective general availability releases,
+we will do our best to rebase/retest `cri-containerd` with these releases on a
+weekly/monthly basis. Similarly, given that `cri-containerd` uses the Open
+Container Initiative (OCI) [image](https://github.com/opencontainers/image-spec)
+and [runtime](https://github.com/opencontainers/runtime-spec) specifications, we
+will also do our best to update `cri-containerd` to the latest releases of these
+specifications as appropriate.
+### Install Dependencies and Build cri-containerd
+1. Install and setup a go1.8.x development environment.
+1. Make a local clone of this repository.
+1. From your github.com/kubernetes-incubator/cri-containerd/ path:
+```shell
+# Note: install.deps installs the above mentioned binary dependencies and cri-tools (discussed below)
+make install.deps
+make
+sudo make install
+```
+### Running with cri-tools
+Another Kubernetes incubator project called [cri-tools](https://github.com/kubernetes-incubator/cri-tools)
+includes programs for exercising CRI implementations such as `cri-containerd`.
+For example, cri-tools includes `critest` which is a testing tool for running
+CRI verification and benchmark tests.
+Use `critest` to validate your installation of `cri-containerd`:
+```shell
+make test-cri
+```
+### Running with Kubernetes
+If you already have a working development environment for Kubernetes, you can
+try cri-containerd in a local cluster:
+
+1. Start `containerd` as root in a first terminal:
+```shell
+sudo containerd
+```
+2. Start `cri-containerd` as root in a second terminal:
+```shell
+sudo cri-containerd -v 2 --alsologtostderr
+```
+3. From the kubernetes project directory startup a local cluster using `cri-containerd`:
+```shell
+KUBERNETES_PROVIDER=local CONTAINER_RUNTIME=remote CONTAINER_RUNTIME_ENDPOINT='/var/run/cri-containerd.sock' ./hack/local-up-cluster.sh
```

Comment:
As is mentioned in https://github.com/kubernetes-incubator/cri-containerd/pull/89#discussion_r124410361, `KUBERNETES_PROVIDER=local` is not required.